.ci_support/linux_64_.yaml

```diff
@@ -1,8 +1,10 @@
 cdt_name:
-- cos7
+- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

Comment:
Worth confirming that Python 3.9 is supported - and I'm not saying it is - since that's what's communicated to the bots here

Reply:
It should be supported -- a quick look shows we didn't add anything that doesn't work with 3.9

Reply:
(also the deps for this package will likely limit the lower python version)